src/kitovu/utils.py

```diff
@@ -24,6 +24,26 @@ def get_password(plugin: str, identifier: str) -> str:
     return password
 
 
+def sanitize_filename(name: str, replacement: typing.Optional[str] = '_') -> str:
+    r"""Replace invalid filename characters.
+
+    Note: This does not escape directory separators (/ and \).
+
+    Args:
+        name: The filename.
+        replacement: The replacement character (or None).
+    """
+    if replacement is None:
+        replacement = ''
```

Comment:
We don't need this for kitovu, so let's remove it. Actually, we don't need the ability to customize the replacement char at all.